playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './tests',
```

Comment:
How do you envision the structure of our test files? How will the file directory look like and where will the test files live? For example, if I have a component named Test.tsx, where will it's test file live in the folder structure?

Reply:
Hey Shashi! After doing some research, 

- for unit testing:
      the test files will have same co-location as the component file it's being tested on
      
- for integration and E2E testing:
      It appears there is a top level directory folder named "playwright" that will consists of the tests.
      
For this line of code, I'd recommend changing the name folder from "tests" to "playwright" to show that it is E2E testing. 

Please let me know what your thoughts are. Thanks!

Reply:
I'd suggest keeping the test files together all in one directory. Giving them different file naming patterns will help determine which file is which. Also, I don't think this path will work for individual tests. I believe this is pointing to a root directory instead of the component directory themselves. Please test this out. Ensure that it will work inside directories and outside. Meaning, if it's named hello.spec.ts and lives inside /components/hello/hello.spec.ts, it should execute.